content/integrations/kubernetes.md

```diff
@@ -5,9 +5,47 @@ kind: integration
 git_integration_title: kubernetes
 ---
 
+# Overview
+
 Get metrics from your Kubelets in real time to:
 
-* Visualize your Kubernetes cluster performance
+* Vizualize your Kubernetes cluster performance
+* Collect performances metrics for your containers, pods, container namespaces
+* Create monitors on the status of your Kubelets
+* Ingest Kubernetes labels as tags in Datadog.
+
+# Installation
+
+The Kubernetes integration is deployed as a docker container along side your existing workloads.
+
+## Installation via DaemonSets (Kubernetes >=1.1.0)
```

Comment:
so should this be more like 1.1.0>=Kubernetes >= 1.2.0? since it no longer applies ?